src/main/conversions.c

```diff
@@ -434,6 +434,7 @@ as_status as_partitions_status_to_pyobject(
             Py_DECREF(new_py_tuple);
```

Comment:
I think py_id needs an XDECREF in the error case

Reply:
Fixed. This was never [covered](https://app.codecov.io/gh/aerospike/aerospike-client-python/pull/375/blob/src/main/conversions.c#L431) in the tests, so a memory leak wasn't reported.